mne/chpi.py

```diff
@@ -292,24 +311,51 @@ def _fit_chpi_pos(coil_dev_rrs, coil_head_rrs, x0):
     return x, 1. - result / denom
 
 
+def _fit_dev_head_trans(dev_pnts, head_pnts):
+    """Compute Device to Head transform allowing for permutiatons of points."""
+    id_quat = np.concatenate([rot_to_quat(np.eye(3)), [0.0, 0.0, 0.0]])
```

Comment:
FYI `rot_to_quat(eye(3)) == np.zeros(3)`, but I don't mind this either because it's explicit